docs/resources/USER.md

```diff
@@ -96,6 +96,9 @@ Modify the requestors user account settings.
 | new_password | string | if passed, will change the users password to this value |
 | avatar | [avatar data](#DOCS_USER/avatar-data) | if passed, modifies the users avatar |
 
+>warn
```

Comment:
Maybe just preface the table with this warning, and have an extra column specifying if a field should be sent for bots or not? I'm unsure if that will look better or not.